pkg/minikube/bootstrapper/images/images.go

```diff
@@ -29,8 +29,12 @@ import (
 // Pause returns the image name to pull for a given Kubernetes version
 func Pause(v semver.Version, mirror string) string {
 	// Should match `PauseVersion` in:
+	// https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/constants/constants.go
+	pv := "3.4.1"
```

Comment:
this means we will need to bump the preload version
https://github.com/medyagh/minikube/blob/1be59b8f23c66d9a6be174dcd579b17302857f5f/pkg/minikube/download/preload.go#L45

lets also comment somewhere in this file that bump preload version if u change this versions

Reply:
I think it only changed for Kubernetes 1.21.0, but maybe you had already generated preloads for that with the wrong images ?

Reply:
Indeed, preloaded-images-k8s-v10-v1.21.0-docker-overlay2-amd64.tar.lz4 has the wrong pause and the wrong coredns

Reply:
the previous versions are all correct though, and I'm not sure we built anything for 1.22 yet ? (I added "master", not alpha)